test/feature/finding_controllers.feature

```diff
@@ -48,30 +48,7 @@ Feature: Finding controllers
           res.end 'user list'
       module.exports = UsersController
       """
-    And an exprestive app with the option "controllersDirPath" set to `"./my_controllers"`
-    When making a GET request to "/users"
-    Then the response code should be 200
-    And the response body should be "user list"
-
-
-  Scenario: an application with a restrictive controller whitelist
```

Comment:
Yeah, now that we're using glob for both `controllersDirPath` and `controllersWhitelist` this test isn't necessary anymore.  It was mainly just testing  `controllersWhitelist` functionality